Module 2 - How do I connect to the cluster.md

```diff
@@ -0,0 +1,47 @@
+# Module 2 - How do I connect to the cluster?
+
+To access the resources of an HPC cluster you need to connect to it through your computer. In this module you will learn how to connect to a cluster, navigate its file system, and run code on the cluster. You will also learn how to transfer data to and from the cluster.
+
+
+# Learning Outcomes
+
+By the end of this module, you will be able to:
+
+- Connect to an HPC cluster using SSH
+- Edit code and run a simple job on the cluster
+- Use a variety of tools to transfer data to and from the cluster
+
+
+# Connecting to the Cluster using SSH
+
+- What is SSH?
+- How to connect to the cluster using SSH?
+- How to set up SSH keys?
```

Comment:
This is a good idea. We can even make use of markdown code-blocks here to have step-by-step commands to copy/paste